core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -23,15 +23,15 @@ import kafka.api.{ApiVersion, KAFKA_0_8_2}
 import kafka.cluster.EndPoint
 import kafka.message._
 import kafka.utils.{CoreUtils, TestUtils}
+import kafka.zk.ZooKeeperTestHarness
 import org.apache.kafka.common.config.ConfigException
 import org.apache.kafka.common.metrics.Sensor
 import org.apache.kafka.common.network.ListenerName
 import org.apache.kafka.common.security.auth.SecurityProtocol
 import org.junit.Assert._
 import org.junit.Test
-import org.scalatest.Assertions.intercept
 
-class KafkaConfigTest {
+class KafkaConfigTest extends ZooKeeperTestHarness {
```

Comment:
It's a little annoying that we had to do this just for the one test case. Maybe we could add a `KafkaServerTest` since `KafkaServer` is where the check was added?